ui/pages/swaps/build-quote/build-quote.js

```diff
@@ -627,44 +627,41 @@ export default function BuildQuote({
                   alt={t('swapSwapSwitch')}
                 />
               </Box>
-              <Typography variant={TypographyVariant.H7} marginTop={0}>
+              <Text variant={TextVariant.bodySm} as="h6" marginTop={0}>
                 {t('stxDescription')}
-              </Typography>
-              <Typography
-                as="ul"
-                variant={TypographyVariant.H7}
-                fontWeight={FONT_WEIGHT.BOLD}
-                marginTop={3}
-              >
-                <li>{t('stxBenefit1')}</li>
-                <li>{t('stxBenefit2')}</li>
-                <li>{t('stxBenefit3')}</li>
-                <li>
-                  {t('stxBenefit4')}
-                  <Typography
-                    as="span"
-                    fontWeight={FONT_WEIGHT.NORMAL}
-                    variant={TypographyVariant.H7}
-                  >
-                    {' *'}
-                  </Typography>
-                </li>
-              </Typography>
-              <Typography
-                variant={TypographyVariant.H8}
+              </Text>
+              <Text as="ul" variant={TextVariant.bodySm} marginTop={3}>
+                <strong>
+                  <li>{t('stxBenefit1')}</li>
+                  <li>{t('stxBenefit2')}</li>
+                  <li>{t('stxBenefit3')}</li>
+                  <li>
+                    {t('stxBenefit4')}
+                    <Text
+                      as="span"
+                      fontWeight={FontWeight.Normal}
+                      variant={TextVariant.bodySm}
+                    >
+                      {' *'}
+                    </Text>
+                  </li>
+                </strong>
+              </Text>
+              <Text
+                variant={TextVariant.bodyXs}
+                as="h6"
                 color={TextColor.textAlternative}
                 boxProps={{ marginTop: 3 }}
               >
                 {t('stxSubDescription')}&nbsp;
-                <Typography
+                <Text
                   as="span"
-                  fontWeight={FONT_WEIGHT.BOLD}
-                  variant={TypographyVariant.H8}
+                  variant={TextVariant.bodyXs}
```

Comment:
you can change the variant to `variant={TextVariant.bodyXsBold}` to save the <strong> tag :)

Reply:
Yes @DDDDDanica , you mentioned this in another PR, I forgot to apply that here.

Thank you.👍

Reply:
Hey @DDDDDanica ,

As @brad-decker mentioned here 
https://github.com/MetaMask/metamask-extension/pull/18752#discussion_r1174597629

This is the same file he suggested the changes in.

Please check.